config/features.yml

```diff
@@ -1232,6 +1232,9 @@ features:
   disability_compensation_prevent_submission_job:
     actor_type: user
     description: If enabled, the submission form526 record will be created, but there will be submission job
+  disability_compensation_remove_pciu:
+    actor_type: user
+    description: If enabled, the submission form526 record will be created, but there will be submission job
```

Comment:
Should we update the description? Looks like it might have been copied from L1234. :)

Reply:
Bah! Sloppy! I'm on it, thanks